modules/op-pipelines-as-code-command-reference.adoc

```diff
@@ -0,0 +1,109 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-command-reference_{context}"]
+= Pipelines as Code command reference 
+
+[role="_abstract"]
+The `tkn-pac` CLI tool offers the following capabilities:
+
```

Comment:
You should add periods to the end of each item in the following bulleted list to be consistent with the list at the beginning.